docs/README_UA.md

```diff
@@ -0,0 +1,541 @@
+  - [Масова індексація] (# об'ємна індексація)
+  - [Фільтрована індексація] (# відфільтрована-індексація)
+  - [Індексація попиту] (# індексація за запитом)
+  - [Скасування вилучення за запитом] (# видалення за запитом)
+  - [Обрізання індексу] (# truncating-an-index)
+  - [Обмеження] (# обмеження)
+    - [Автоіндексація] (# автоматична індексація)
+    - [Пошук відразу після es-індексованої події] (# search-відразу-після-es-indexed-event)
+- [Картографування] (# відображення)
+  - [Географічне відображення] (# географічне відображення)
+    - [Індексація геоточки] (# індексація-гео-точка)
+    - [Індексація геофігури] (# індексація-гео-форма)
+  - [Створення відображень за запитом] (# створення-відображення на вимогу)
+- [Запити] (# запитів)
+  - [Гідратація] (# гідратація)
+
+## Встановлення
+
+Остання версія цього пакета буде максимально наближена до найновіших пакетів `elasticsearch` та` mongoose`.
+
+`` `баш
+npm встановити -S монгоосастичний
+`` `
+
+## Налаштування
+
+### Model.plugin (монгоосастика, параметри)
+
+Варіанти:
+
+* `index` - індекс у Elasticsearch для використання. За замовчуванням плюралізація назви моделі.
+* `type` - тип, який ця модель представляє в Elasticsearch. За замовчуванням назва моделі.
+* `esClient` - існуючий екземпляр Elasticsearch` Client`.
+* `hosts '- масив хостів Elasticsearch працює на.
+* `host` - хост Elasticsearch працює на
+* `port` - порт Elasticsearch працює на
+* `auth` - автентифікація, необхідна для досягнення сервера Elasticsearch. У стандартному форматі "ім'я користувача: пароль"
+* `Protocol '- протокол, який використовує сервер Elasticsearch. За замовчуванням http
+* `гідрат '- чи потрібно шукати результати в mongodb раніше
+* `hydrateOptions '- параметри переходу у гідратну функцію
+* `bulk` - параметри розміру та затримки для масової індексації
+* `filter` - функція, яка використовується для відфільтрованої індексації
+* `перетворення '- функція, що використовується для перетворення серіалізованого документа перед індексуванням
+* `populate '- масив об'єктів параметрів для заповнення масиву мангустів
+* `indexAutomatically '- дозволяє відключити індексацію після збереження моделі, коли вам потрібен більш тонкий контроль, коли документи індексуються. Значення за замовчуванням - справжнє
+* `customProperties` - об'єкт, що деталізує додаткові властивості, які будуть об'єднані на відображення за замовчуванням типу, коли викликається` createMapping '.
+* `saveOnSynchronize` - запускає метод мангуста збереження (і попереднього збереження) при синхронізації колекції / індексу. Значення за замовчуванням - справжнє
+
+
+Щоб модель була індексована в Elasticsearch, просто додайте плагін.
+
+`` javascript
+var mongoose = вимагати ('мангуста')
+  , mongoosastic = вимагати ('mongoosastic')
+  , Схема = мангуста.Схема
+
+var User = нова схема ({
+    назва: Рядок
+  , електронна пошта: String
+  , місто: Рядок
+})
+
+User.plugin (монгоосастичний)
```

Comment:
This is not valid JavaScript code. The code example should be always valid. Add line comments if you feel it would be more clear.